test/unit/resources/azure_express_route_providers_test.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'helper'
+require 'azure_express_route_providers'
+
+class AzureExpressRouteServiceProvidersConstructorTest < Minitest::Test
+  def test_resource_type_not_ok
+    assert_raises(ArgumentError) { AzureVirtualMachines.new(resource_provider: 'some_type') }
```

Comment:
this is pointing to a wrong Test Class name

Reply:
Yes Sateesh making that change.